test/external/external_test_hsa_backend.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+# import subprocess
+from tinygrad.tensor import Tensor, dtypes
+import numpy as np
+
+
+class TestHSABackend(unittest.TestCase):
```

Comment:
Huh, why is this the name?

Also, this test isn't running in CI

Reply:
was to regression test issue #3536 that only occurred with "HSA=1"
moved the tests to CI

I noticed these tests caused segfault on LLVM backend, so only testing on HIP now